cookbook/views/api.py

```diff
@@ -282,6 +282,10 @@ def get_queryset(self):
         queryset = ViewLog.objects.filter(created_by=self.request.user).all()[:5]
         return queryset
 
+class NutritionInformationViewSet(viewsets.ModelViewSet):
```

Comment:
if there is no call to this api anyway as the information is always returned with the recipe there is no need to add the ViewSet. 

If we want to have a seperate API endpoint for just nutritional information we should add the appropriate permission classes and filters

Reply:
I removed it